src/components/builder/comparison-value-input.tsx

```diff
@@ -0,0 +1,60 @@
+import {useFormikContext} from 'formik';
+import {ExtendedComponentSchema, Utils as FormioUtils} from 'formiojs';
+import {useContext, useEffect, useState} from 'react';
+import usePrevious from 'react-use/esm/usePrevious';
+
+import {BuilderContext} from '@/context';
+import {getRegistryEntry} from '@/registry';
+import Fallback from '@/registry/fallback';
+import {comparisonValueComponentProps} from '@/registry/types';
+
+export interface simpleConditional {
+  conditional: {
+    eq: string;
+    when: string;
+    show: null | boolean;
+  };
```

Comment:
this doesn't exactly line up with the formio type definition: https://github.com/formio/formio.js/blob/bebc2ad73cad138a6de0a8247df47f0085a314cc/types/components/schema.d.ts#L157, which we rely on (semi-implicitly) via https://github.com/open-formulieren/types/blob/5a83f493525aeeb9ca5bca6386429853479a6536/src/formio/base.ts#L106

Most notably:

* `show` is not allowed to be `null`, it needs to be `bool` or undefined
* `json` is not specified, but since we don't make use of it, okay to leave this out.

I'd probably define this as:

```ts
import type { ConditionalOptions } from 'formiojs';

export interface SimpleConditional {
  conditional: Omit<ConditionalOptions, 'json'>;
}
```
to avoid type checker issues in the future.

I'm also not sure about making `conditional` a required property - the upstream type has it as optional: https://github.com/formio/formio.js/blob/bebc2ad73cad138a6de0a8247df47f0085a314cc/types/components/schema.d.ts#L87